localization/english/event_localization/wc_major_decisions_pandaria_unification_l_english.yml

```diff
@@ -0,0 +1,78 @@
+﻿l_english:
+ pandaria_unification.0001.t:0 "The Mogu Empire Reborn"
+ pandaria_unification.0001.desc.intro:0 "Thousands of [GetTrait('creature_mogu').GetName( GetNullCharacter )] shout in unison as [scoped_emperor.GetName] is proclaimed Emperor of the renewed [GetTitleByKey('e_pandaria_empire').GetName]. The entire continent of Pandaria is now once again united under the Mogu yoke. The Empire, once founded by [thunderking.GetFirstNameNicknamed], has now been reborn."
+ pandaria_unification.0001.desc.mogu:0 "\n\nOur people have reclaimed their rightful place as the rulers of the world. The dawn of a new age is upon us, one marked by might and conquest!"
+ pandaria_unification.0001.desc.neutral:0 "\n\nFor the other races of Pandaria, this will surely mark a time of strife and tyranny, enslaved by the cruel Mogu."
+ pandaria_unification.0001.mogu:0 "Our time has come!"
+ pandaria_unification.0001.neutral:0 "And none who dared and fought could stand..."
+ 
+ pandaria_unification.0002.t:0 "Reunited Pandaren Empire"
+ pandaria_unification.0002.desc:0 "In a grand ceremony, [scoped_emperor.GetName] is proclaimed the ruler of the renewed [GetTitleByKey('e_pandaria_empire').GetName]. Pandaria is now united once more under the guiding hand of a [GetTrait('creature_pandaren').GetName( GetNullCharacter )] ruler, ready to face whatever outside threats await."
+ pandaria_unification.0002.pandaren:0 "An era of peace awaits."
```

Comment:
would it maybe be a bit more flavorful a la chinese culture to say something like "ten thousand years" vs an era? https://en.wikipedia.org/wiki/Ten_thousand_years it's like, a big thing

Reply:
Done! It fits quite well as it was roughly 12.000 years canonically which fits still